README.md

```diff
@@ -156,8 +155,7 @@ variables:
    sf:part_type = "part_type" ; // Variable indicating if parts are holes or not -- not required unless polygons with holes are present.
    sf:node_coordinates = "x y" ; // variables containing spatial node data.
  int node_count(instance); // count of coordinates in each instance geometry
- int part_node_count(part) ; // count of coordinates in each geometry part
- int part_type(part) ; // type of each geometry part
+ int part_count(instance) ; // count of parts belonging to each geometry
```

Comment:
hah... not thinking straight on this one. We really do need the part dimension then.

So the question is actually what you typed and I didn't read correctly over in #53. I think counting the same thing (nodes) in each is going to lead to less confusion. So never mind on this PR.